src/dragtor/index/chunk.py

```diff
@@ -68,13 +69,46 @@ def chunk_and_annotate(self, text: str) -> tuple[list[str], list[tuple[int, int]
 
         return chunks, span_annotations
 
+class LangchainChunker(Chunker):
```

Comment:
Langchain provides multiple different chunkers, so this name is misleading. I would prefer something like `RecursiveCharacterChunker`